src/components/editor/editor.js

```diff
@@ -2925,6 +2925,10 @@ Editor.prototype = {
     this.destroyPreviewMode();
     this.element.addClass('is-disabled').attr('contenteditable', 'false');
     this.container.addClass('is-disabled');
+
+    setTimeout(() => {
```

Comment:
```suggestion
    setTimeout(() => {
```

Is this timeout really needed?

Reply:
When I invoked the disable function, it doesn't work. I suspected that it calls before when you really needed it.